queries.sql

```diff
@@ -38,4 +38,71 @@ SELECT species, AVG(escape_attempts) FROM animals GROUP BY species;
 
 SELECT species, AVG(escape_attempts) FROM animals 
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+-- Querying joined tables
+SELECT animals.name AS animal_name, owners.full_name AS owners_name FROM animals 
+JOIN owners ON animals.owner_id= owners.id 
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name AS animal_name, species.name as species_name FROM animals JOIN species 
+ON animals.species_id= species.id WHERE species.name = 'Pokemon';
+
+SELECT DISTINCT owners.full_name AS owners_name, animals.name AS animal_name FROM animals 
+RIGHT JOIN owners ON animals.owner_id = owners.id;
+
+SELECT species.name AS animal_species, COUNT(animals.name) as total FROM animals JOIN species 
+ON animals.species_id = species.id GROUP BY species.name;
+
+SELECT owners.full_name AS owners_name, animals.name AS animal_name, species.name AS species_name 
+FROM animals JOIN species ON animals.species_id = species.id 
+JOIN owners ON animals.owner_id = owners.id  
+WHERE owners.full_name='Jennifer Orwell' AND species.name = 'Digimon';
+
+SELECT owners.full_name AS owners_name, animals.name AS animal_name, animals.escape_attempts FROM animals  
+JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts =0; 
+
+SELECT owners.full_name AS owners_name, COUNT(animals.name) AS animal_count FROM animals  
+JOIN owners ON animals.owner_id = owners.id GROUP BY owners.full_name ORDER BY COUNT(animals.name) DESC LIMIT 1; 
+
+-- Multi joined table queries 
+SELECT animals.name AS animal_name, vets.name AS vet, visits.visit_date AS visit_data 
+FROM animals JOIN vets ON animals.owner_id = vets.id 
+JOIN visits ON visits.vets_id = animals.owner_id 
+WHERE vets.name = 'William Tatcher' ORDER BY visits.visit_date DESC LIMIT 1;
+
+SELECT vets.name, COUNT(DISTINCT visits.animals_id) FROM vets 
+JOIN visits ON vets.id = visits.vets_id WHERE vets.name ='Stephanie Mendez' 
+GROUP BY vets.name;
+
+SELECT vets.name AS vet_name, species.name AS species_name, specializations.vets_id AS specialization_id FROM vets 
+LEFT JOIN specializations ON vets.id = specializations.vets_id LEFT JOIN species 
+ON species.id = specializations.species_id;
+
+SELECT animals.name AS animal_name, visits.visit_date AS visit_date, vets.name AS vet_name 
+FROM animals JOIN visits ON animals.id = visits.animals_id JOIN vets ON vets.id = visits.vets_id 
+WHERE visit_date BETWEEN '2020-04-01' AND '2020-08-30' ORDER BY visit_date DESC;
```

Comment:
- [ ] I think there is a condition for this query  **this query List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020** I would like to recommend you add for example `WHERE vets.name = 'Stephanie Mendez'`  and please make sure that all your queries correspond to project requirement and don't forget to add more screenshots 👍